rmq/consumer/config.go

```diff
@@ -60,6 +60,30 @@ func DefaultConfig(name string) Config {
 	}
 }
 
+// ExchangeConfig is the configuration for exchange creation.
```

Comment:
where do you define the default behaviour?

Reply:
Default behaviour is defined through the use of the `DefaultExchangeConfig( )` helper function, which requires only 2 params and generates a ExchangeConfig( ) object. If users do not use the helper, they will have to specify each field in the ExchangeConfig manually.